mrtt-ui/src/components/ProjectDetailsForm/ProjectDetailsForm.js

```diff
@@ -190,8 +175,15 @@ function ProjectDetailsForm() {
           />
           <div className={styles.invalid}>{errors.countries?.message}</div>
         </div>
-        <Button sx={{ marginTop: '1em' }} variant="contained" type="submit">
-          Submit
+        {/* Draw Pologon - TO BE INSERTED */}
+        <div className={styles.formGroup}>
+          <FormLabel sx={{ color: 'black', marginBottom: '1.5em' }}>
+            1.3 What is the overall site area?
+          </FormLabel>
+        </div>
+        {isError && <div className={styles.invalid}>{'Submit failed, please try again'}</div>}
```

Comment:
This could be simplified to
```suggestion
        {isError && <div className={styles.invalid}>Submit failed, please try again</div>}
```

Reply:
ah yes, no idea why those curly braces got added